src/PKSim.Presentation/Presenters/Individuals/CreateIndividualPresenter.cs

```diff
@@ -92,9 +90,58 @@ public override void WizardNext(int previousIndex)
          base.WizardNext(previousIndex);
       }
 
-      public PKSim.Core.Model.Individual Individual
+      public Individual Individual
       {
          get { return PresenterAt(IndividualItems.Settings).Individual; }
       }
+
    }
-}
\ No newline at end of file
+
+   public class CreateIndividualPresenter : AbstractCreateIndividualPresenter
+   {
+      public CreateIndividualPresenter(ICreateIndividualView view, ISubPresenterItemManager<IIndividualItemPresenter> subPresenterItemManager,
+         IDialogCreator dialogCreator,
+         IBuildingBlockPropertiesMapper propertiesMapper, IObjectBaseDTOFactory buildingBlockDTOFactory)
+         : base(view, subPresenterItemManager, dialogCreator, propertiesMapper, buildingBlockDTOFactory, IndividualItems.All)
+      { }
+
+      protected override void UpdateControls(int indexThatWillHaveFocus)
+      {
+         UpdateViewStatus();
+         _view.NextEnabled = PresenterAt(IndividualItems.Settings).CanClose && indexThatWillHaveFocus != IndividualItems.Expression.Index;
+         _view.OkEnabled = CanClose;
+         _view.SetControlEnabled(IndividualItems.Expression, PresenterAt(IndividualItems.Settings).IndividualCreated);
+         _view.SetControlEnabled(IndividualItems.Parameters, PresenterAt(IndividualItems.Settings).IndividualCreated);
+      }
+   }
+   }
+
+//try moving to the Starter namespace
+   public class CreateIndividualPresenterForMoBi : AbstractCreateIndividualPresenter
```

Comment:
I am wondering why this is not just a simple flag that says Expression supported or not? RIght now it's an exact copy of the other code without the last tab. All those expressions such as
```
_view.SetControlEnabled(IndividualItems.Expression, PresenterAt(IndividualItems.Settings).IndividualCreated);
         _view.SetControlEnabled(IndividualItems.Parameters, PresenterAt(IndividualItems.Settings).IndividualCreated);
`` can be rewritten to use sthg like all except first

Reply:
The idea was to reduce the number of registrations required. We didn't want to register the presenter for Expressions and all its dependencies if it's not going to be shown.

Reply:
Hum. registration costs nothing. Instantiation is. Do you register separately based on the use case? I can't seem to see a difference in the registration code